admin/assets/stylesheets/elements/_not-taking-check.scss

```diff
@@ -24,3 +24,21 @@
     }
   }
 }
+
+#pupilsList {
+  th:nth-child(1),
```

Comment:
nth-child(x) is not supported in IE8, but then these tweaks are minor, they just enhance a little bit the table view, I could remove them completely and it would still be OK.